.circleci/config.yml

```diff
@@ -209,104 +329,150 @@ jobs:
               Write-Host "Forcing build failure due to test failure(s) :'("
               $host.SetShouldExit(1)
             }
+
   publish:
-    <<: *defaults
+    parameters:
+      e:
+        type: executor
+    executor: << parameters.e >>
     steps:
-      - add_ssh_keys: *ssh-config
+      - slack/notify:
+          channel: 'pdt_releases'
+          color: '#FFDEAD'
+          message: 'Starting publish for VSCode Extensions'
+      - ssh-config
       - checkout
-      - restore_cache: *restore_cache
+      - gus-prepare-environment-variables
+      - restore_dependency_cache
       - run:
           name: 'Setup gus case management'
           command: |
             sudo npm install -g @salesforce/change-case-management
-      - run: *gus-prepare-environment-variables
+      - sfchangecase/install
```

Comment:
We need to remove this until we merge a fix to the case management orb. We currently replaced this line with "manually" installing `@salesforce/change-case-management` above.